### PR TITLE
Address assorted Weblate comments

### PR DESCRIPTION
**Pull request by Frenzie**
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7154)

### CODE REVIEW

**Comment by poire-z** [comment]
> Change KB/MB/GB to KiB/MiB/GiB since that's what it is
> I'd probably prefer kB/MB/GB instead though. I.e., per 1000 instead of 2^10.

So do I. I never ever see MiB, which just feels pedantic.
So please keep them (and their existing translations) and fix the computation :)

**Review by NiLuJe — APPROVED** [review]

**Review by poire-z — APPROVED** [review]

File: frontend/ui/elements/screen_android.lua [hunk 1]
```diff
@@ -26,9 +27,9 @@ local timeout_custom7 = 30 * 60 * 1000
 local function humanReadableTimeout(timeout)
     local sec = timeout / 1000
     if sec >= 120 then
```
Comment (pazos) [inline frontend/ui/elements/screen_android.lua:29]: `if sec >= 60 then`

File: plugins/calibre.koplugin/wireless.lua [hunk 2]
```diff
@@ -617,7 +618,7 @@ function CalibreWireless:deleteBook(arg)
             if i == 1 then
                 msg = T(_("Deleted file: %1"), BD.filepath(arg.lpaths[1]))
             else
-                msg = T(_("Deleted %1 files in %2:\n %3"),
+                msg = T(N_("Deleted 1 file in %2:\n %3", "Deleted %1 files in %2:\n %3", #arg.lpaths),
```
Comment (pazos) [inline plugins/calibre.koplugin/wireless.lua:621]: single files are covered by `i == 1`. I guess you can remove that.

File: plugins/calibre.koplugin/wireless.lua [hunk 3]
```diff
@@ -617,7 +618,7 @@ function CalibreWireless:deleteBook(arg)
...
                     #arg.lpaths, BD.filepath(inbox_dir), titles)
```
Comment (pazos) [inline plugins/calibre.koplugin/wireless.lua:622]: and remove `#arg.lpaths` from here

File: frontend/ui/elements/screen_android.lua (hunk 1 above)
Comment (Frenzie) [inline frontend/ui/elements/screen_android.lua:29]:
```suggestion
    if sec >= 60 then
```

**Pull request merged**